tests/utils/test_config.py

```diff
@@ -93,3 +94,76 @@ def test_from_configuration_files_get_typed_value(tmp_path_factory: pytest.TempP
 
     assert Config().get_bool("legacy-current-snapshot-id")
     assert Config().get_int("max-workers") == 4
+
+
+@pytest.mark.parametrize(
+    "config_setup, expected_result",
+    [
+        # Validate lookup works with: config > home > cwd
+        (
```

Comment:
nit: for test readability, use `["PYICEBERG_HOME", "HOME", and "CURRENT"]`
and replace `both` with a list ["HOME", "CURRENT"]

Reply:
looks like the parameterize test is testing
1. PYICEBERG_HOME
2. HOME
3. CURRENT
4. None
5. "both" / ["HOME", "CURRENT"]

i'd add a test for all 3